src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -488,7 +488,7 @@ public void AddClaims(IEnumerable<Claim> claims)
             }
         }
 
-        internal static object GetClaimValueUsingValueType(Claim claim)
+        private static object GetClaimValueUsingValueType(Claim claim)
```

Comment:
I am concerned this is a breaking change for users. Is there a specific issue that this is causing?